### PR TITLE
Add @brian-umberger's contact tracking to example2DWalking.

### PR DESCRIPTION
**Pull request by chrisdembia**
Fixes issue #581 

### Brief summary of changes

- This PR incorporates MocoContactTrackingGoal into example2DWalking, thanks to @brian-umberger.

### CHANGELOG.md (choose one)

- [x] updated

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-moco/594)

### CODE REVIEW

File: Moco/Bindings/Java/Matlab/Examples/example2DWalking/example2DWalking.m [hunk 1]
```diff
@@ -50,26 +51,36 @@
...
 track = MocoTrack();
 track.setName('gaitTracking');
 
+% Set the weights for the terms in the objective function.
+%
+% Note: If GRFTrackingWeight is set to 0 then GRFs will not be tracked. Setting
+% GRFTrackingWeight to 1 will cause the total tracking error (states + GRF) to
+% have about the same magintude as control effort in the final objective value.
+controlEffortWeight = 10;
+stateTracingkWeight = 1;
```
Comment (antoinefalisse) [inline Moco/Bindings/Java/Matlab/Examples/example2DWalking/example2DWalking.m]: There is a typo here.

**Comment by brian-umberger** [comment]
Thanks - good eyes. It looks like the typo is consistent throughout, so
does not affect the results, but it should be corrected to:
stateTrackingWeight (2 occurrences).

Brian

File: Moco/Bindings/Java/Matlab/Examples/example2DWalking/example2DWalking.m (hunk 1 above)
Comment (chrisdembia) [reply Moco/Bindings/Java/Matlab/Examples/example2DWalking/example2DWalking.m]: Fixed

**Comment by chrisdembia** [comment]
Thanks @antoinefalisse and @brian-umberger; I fixed the typos.

**Comment by chrisdembia** [comment]
Thanks @nickbianco

**Pull request merged**